coursemapper-kg/recommendation/app/services/course_materials/recommendation/youtube_service.py

```
@@ -42,17 +43,65 @@
         api_version,
         developerKey="AIzaSyClxnNwQ1x34pGioQazLlGxOjO9Fp2GGTY",
     )
+    DEVELOPER_KEYS = [  
+                        "AIzaSyBphZOn7EJmPMmZwrB71aepaA5Rbuex9MU",
+                        "AIzaSyADNntK6m7DbA6eZFYOa9Y8e6IYHykUUFE",
+                        "AIzaSyD_CGmR_Voq4DIV5okRaR6G8adoe-ZSZsM",
+                        "AIzaSyClxnNwQ1x34pGioQazLlGxOjO9Fp2GGTY",
+                        "AIzaSyB2Wck31LUlgsqI7dgTcC2dMeeVXgb9TDI"
+                    ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)